usagov/vote-gov-nvrf-app
# VOTE-1947 Reorganize repo to follow react best practices

Jira ticket: https://cm-jira.usa.gov/browse/VOTE-1947

Testing steps:

1. Test the form with the following options for Alaska (all registration options template)
- Registration option = National Voter Registration Form Filler
- Registration Path = change registration
- Fill out all fields, including optional
- On confirmation page, go back to Personal Info section, change phone number, confirm the change takes effect
- On delivery page, test both the "open in a new tab" and the "download" function. Confirm data is applied to the PDF correctly and instruction pages are correct

2. Test the registration options page for each of these states/types (you do not have to test the whole form again)
- Arkansas (all mail-in options)
- Guam (online, no mail options)
- New Hampshire (state mail, no NVRF)
- Wisconsin (online, state mail, no NVRF)
- North Dakota (no registration)
- American Samoa (in person only)
